src/apibuilder-cli/git.rb

```diff
@@ -2,9 +2,26 @@ module ApibuilderCli
 
   module Git
 
+    def Git.current_branch
+      `git rev-parse --abbrev-ref HEAD`.strip
+    end
+
+    def Git.generate_version(commits_back = nil, legacy = false)
+      raw_version = Git.safe_describe(commits_back)
+      if raw_version == ""
+        ""
+      elsif legacy
+        raw_version
+      else
+        branch = Git.current_branch.downcase
+        suffix = branch == "master" ? "" : "-#{branch}"
```

Comment:
nit: would prefer avoiding the "" here, e.g.:
```
if branch == "master"
  raw_version
else
  "#{raw_version}-#{branch}"
end

Reply:
Will do